src/CommonITILValidation.php

```diff
@@ -666,11 +666,26 @@ public static function getNumberToValidate($users_id)
         global $DB;
 
         $row = $DB->request([
-            'FROM'   => static::getTable(),
+            'FROM'   => Ticket::getTable(),
             'COUNT'  => 'cpt',
             'WHERE'  => [
-                'status'             => self::WAITING,
-                'users_id_validate'  => $users_id
+                [
+                    'id' => new QuerySubQuery([
+                        'SELECT' => 'tickets_id',
+                        'FROM'   => self::getTable(),
+                        'WHERE'  => [
+                            'status' => self::WAITING,
+                            'users_id_validate'  => $users_id,
+                        ]
+                    ])
+                ],
+                'NOT' => [
+                    'AND' => [
+                        [
+                            'status' => Ticket::getClosedStatusArray()
+                        ]
+                    ]
+                ]
```

Comment:
It should do the same
```suggestion
                'NOT' => [
                    'status' => Ticket::getClosedStatusArray(),
                ],
```